_includes/js/queries.md

```diff
@@ -391,3 +424,112 @@ mainQuery.find()
     // There was an error.
   });
 </code></pre>
+
+## Aggregate
+
+Queries can be made using aggregates, allowing you to retrieve objects over a set of input values. The results will not be `Parse.Object`s since you will be aggregating your own fields
+
+* Note: MasterKey is Required. Parse-Server 2.7.1+
+
+Aggregates use stages to filter results by piping results from one stage to the next.
+
+You can create a pipeline using an Array or an Object.
+
+The following example is a pipeline similar to `distinct` grouping by name field.
+
+<pre><code class="javascript">
+// Pipeline object
+const pipeline = {
+  group: { objectId: '$name' }
+ };
+
+// Pipeline array
+const pipeline = [
+  { group: { objectId: '$name' } }
+];
+</code></pre>
+
+For a list of available operators please refer to [Mongo Aggregate Documentation](https://docs.mongodb.com/v3.2/reference/operator/aggregation/).
+
+* Note: Most operation in Mongo Aggregate Documentation will work with Parse-Server, but `_id` doesn't exist. Please replace with `objectId`.
+
+<pre><code class="javascript">
+// Can apply $sum, $avg, $max, $min
+const pipeline = [
+  group: { objectId: null, total: { $sum: '$score' } }
+];
+var query = new Parse.Query("User");
+query.aggregate(pipeline);
+query.find()
+  .then(function(results) {
+    // results contains sum of score field and stores it in results[0].total
+  })
+  .catch(function(error) {
+    // There was an error.
+  });
+</code></pre>
+
+<pre><code class="javascript">
```

Comment:
I would add some descriptive text before this code block to detail what's in it, especially since it's only preceded by another block.